docs/archive-creation.md

```diff
@@ -46,7 +46,8 @@ jobs:
 | `NPM_REGISTRY_DOMAIN` | `"https://npm.pkg.github.com/"`                               | Domain of the private npm registry                                                             |
 | `PACKAGE_MANAGER`     | `"yarn"`                                                      | Package manager with which the dependencies should be installed (`npm` or `yarn`)              |
 | `COMPOSER_ARGS`       | `'--no-dev --no-scripts --prefer-dist --optimize-autoloader'` | Set of arguments passed to Composer                                                            |
```

Comment:
```suggestion
| `COMPOSER_ARGS`       | `'--no-dev --no-scripts --prefer-dist --optimize-autoloader'` | Set of arguments passed to Composer when gathering production dependencies |
```
Please ensure to align the table.

Reply:
I did reformat using the PHPStorm inspection, but yeah I forgot to update the description for `COMPOSER_ARGS`. Thanks for that!